scenes/game/local_game.gd

```diff
@@ -4381,8 +4447,12 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			performing_player.ignore_push_and_pull -= 1
 			if performing_player.ignore_push_and_pull == 0:
 				_append_log_full(Enums.LogType.LogType_Effect, performing_player, "no longer ignores pushes and pulls.")
+		"look_at_top_deck":
```

Comment:
can you just make this "reveal_topdeck" to go with reveal_hand and reveal_hand_and_topdeck?
And put it down there of course

Reply:
actually, i guess there's a slight difference in that reveal goes to both players but look at is just one?
Double check i guess that it makes sense

Reply:
nah i think they were both literally using the same function